literature.bib

```diff
@@ -9942,7 +10008,7 @@ @article{CDS:TSE08
 	address = Washington,
 	tt-tags = {},
 	tc-tags = {},
-	sampling-tags = {},
+	sampling-tags = {name=AETG-SAT, testing efficiency, sampling efficiency, greedy, compares four variations of AETG-SAT, CCIT, testing, feature model,t-wise coverage, unavailable tool, evaluation}
```

Comment:
please always use order as on the website (or in other entries)

Reply:
I will :)